ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpServiceParameterizedTest.java

```diff
@@ -39,17 +38,11 @@ public JsonRpcHttpServiceParameterizedTest(final String requestJson) {
     this.requestJson = requestJson;
   }
 
-  @BeforeClass
-  public static void setup() throws Exception {
+  @Before
```

Comment:
The tests were interfering with each other, by using the same setup for all of them. Now every test has their own isolated setup.